### ISSUE TITLE
Correct builder documentation

### ISSUE THREAD

**Issue by importhuman** [buildpacks/docs#415]
## Summary

The [concepts page](https://buildpacks.io/docs/concepts/#what-is-a-builder) on the docs says this:
> Builders are an ordered combination of buildpacks with a base build image and a **run image**.

While the [builders page](https://buildpacks.io/docs/concepts/components/builder/) has this:
> A builder is an image that contains all the components necessary to execute a build. A builder image is created by taking a build image and adding a lifecycle, buildpacks, and files that configure aspects of the build including the buildpack detection order and the location(s) of the run image

So the *run image* should be replace with "lifecycle", unless I'm misunderstanding the concept.

## Additional Information

Can be labeled as good first issue

**Comment by SanjaySinghRajpoot** [comment]
I would like to work on this issue.

**Comment by importhuman** [comment]
@SanjaySinghRajpoot thank you for your interest! As far as I understand, the issue is valid and hence we should update our documentation, so go ahead! Feel free to reach out to us if you need help either here or on our [Slack](https://slack.buildpacks.io/). 

Tagging the maintainers just in case this issue is incorrect @buildpacks/learning-maintainers

**Comment by jromero** [comment]
I agree with you @importhuman. The builder definitely doesn't have a `run image`. It has a reference to one but it needs to be clarified.

What about something like:

> Builders are an ordered combination of buildpacks with a base build image, **a lifecycle, and reference to a run image**.

Open to better suggestions.

**Comment by SanjaySinghRajpoot** [comment]
ok, I will make appropriate changes to both the concepts page and builders page.
